tests/casp/filesystem_ro.pm

```diff
@@ -16,12 +16,8 @@ use testapi;
 
 sub run() {
     assert_script_run "! touch /should_fail";
-
-    # /etc is ro now but may be changed later
-    assert_script_run "! touch /etc/should_fail";
-
+    assert_script_run "touch /etc/should_succeed";
```

Comment:
Added fate link to test summary. It is also discussed in http://thenewstack.io/micro-os-suses-answer-container-os/ - but I don't want to reference non-suse articles in commit messages :)